cmd/manager/scap.go

```diff
@@ -213,23 +215,28 @@ func getResourcePaths(ds *utils.XMLDocument, profile string) []string {
 	return out
 }
 
-func (c *scapContentDataStream) FetchResources() error {
-	found, err := fetch(c.client, c.resources)
+func (c *scapContentDataStream) FetchResources() ([]string, error) {
```

Comment:
You should also need to update the FetchResources interface (surprised that build did not catch this)

Reply:
Good catch, let me update it.